lib/http2/connection.c

```diff
@@ -991,6 +991,9 @@ static int handle_settings_frame(h2o_http2_conn_t *conn, h2o_http2_frame_t *fram
             *err_desc = "invalid SETTINGS frame (+ACK)";
             return H2O_HTTP2_ERROR_FRAME_SIZE;
         }
+        if (conn->timestamps.settings_acked_at.tv_sec == 0 && conn->timestamps.settings_sent_at.tv_sec != 0) {
+            gettimeofday(&conn->timestamps.settings_acked_at, NULL);
```

Comment:
How about using `h2o_gettimeofday` instead? It is faster, and provides sufficient accuracy for measuring RTT.

Reply:
>How about using `h2o_gettimeofday` instead? It is faster, and provides sufficient accuracy for measuring RTT.

In my local testing `h2o_gettimeofday` was more likely to return the same value for both measurements (yielding an RTT of 0) because it does not update as often, but you are probably right that for production use that will not be a problem.

Reply:
Done in ff9b7f8f1, thank you.